test/Sign.Cli.Test/Options/OptionTests.cs

```diff
@@ -100,20 +111,29 @@ protected virtual void VerifyEqual(T? expectedValue, T? actualValue)
             Assert.Equal(expectedValue, actualValue);
         }
 
-        protected void VerifyHasError(string commandLine)
+        protected void VerifyHasErrors(string commandLine, params string[] expectedErrorMessages)
         {
             ParseResult result = Parse(commandLine);
+            HashSet<string> expectedMessages = new(expectedErrorMessages, StringComparer.Ordinal);
```

Comment:
Is there any case where ordering of error message matters? If yes, then we're losing ordering here.

Reply:
Nope, ordering doesn't really matter.